File: crates/dashboard-model/src/victron_controller/dashboard/from_0_1_0_command_ack.rs


pub fn convert__command_ack__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::command_ack::CommandAck) -> crate::victron_controller::dashboard::command_ack::CommandAck {
    crate::victron_controller::dashboard::command_ack::CommandAck {
        accepted: from.accepted.clone(),
        error_message: from.error_message.clone(),
    }
}

File: web/src/model/victron_controller/dashboard/v0_2_0/from_0_1_0_command-ack.ts
// @ts-nocheck
import {CommandAck as v0_1_0_CommandAck} from '../v0_1_0/CommandAck'
import {CommandAck as v0_2_0_CommandAck} from './CommandAck'

export function convert__command_ack__from__0_1_0(from: v0_1_0_CommandAck): v0_2_0_CommandAck {
    return new v0_2_0_CommandAck (
        from.accepted,
        from.error_message
    )
}